lib/site-inspector/checks/accessibility.rb

```diff
@@ -0,0 +1,62 @@
+require 'json'
+require 'open3'
+require 'mkmf'
+
+class SiteInspector
+  class Endpoint
+    class Accessibility < Check
+      
+      def section508
+        pa11y(:section508)
+      end
+      
+      def wcag2a
+        pa11y(:wcag2a)
+      end
+      
+      def wcag2aa
+        pa11y(:wcag2aa)
+      end
+      
+      def wcag2aaa
+        pa11y(:wcag2aaa)
+      end
+      
+      private
+      
+      def pa11y_installed?
+        !`which pa11y`.empty?
+      end
+      
+      def pa11y(standard)
+        if pa11y_installed?
+          standards = {
```

Comment:
I'd make this a class constant, rather than burying it within a method. I'd also expose a `standard?` both for our own use, and for users to check if we support a given standard.